node-deb

```diff
@@ -555,5 +573,6 @@ if [ "$no_md5sums" -eq 0 ]; then
 fi
 
 log_info 'Building Debian package'
-dpkg-deb --build "$deb_dir" > '/dev/null'
+chmod -R u+rwX,g+rX-w,o+rX-w "$deb_dir"
```

Comment:
I would change this to `chmod 755` just because I find that to be a lot more clear and using the `+` and `-` operators to manually flip all of them.